WcaOnRails/app/views/server_status/index.html.erb

```diff
@@ -58,11 +58,8 @@
     </div>
   </div>
 
-  <div>
-    PID <%= Process.pid %>
-  </div>
+  <%= alert :info, "PID #{Process.pid}" %>
+  <%= alert :info, "Uptime #{duration_to_s(Time.now - WcaOnRails::BOOTED_AT)}" %>
```

Comment:
I think separate, colored alerts for these little pieces of information is a bit much. Just an idea: maybe a [bootstrap well](http://getbootstrap.com/components/#wells) with all these little things together?

Your call, I don't feel too strongly about this.